### PR TITLE
Issue 148

### PR DESCRIPTION
**Pull request by VAO11**
@womenhackfornonprofits/whfnp
## Linked Jira/Github Issue Ticket

Issue #148 
## What are the changes included in this PR?

Replaces the sass with bootstrap.  Removed the redundant content
## If the changes are visual, provide a screenshot.

_Completed Desktop View:_
whfnp_projs-completed_ 148

_Current Desktop View:_
whfnp_projs-current_ 148

_Completed Mobile View:_
whfnp_mob-projs_ 148

_Current Mobile View:_
whfnp_mob2-projs_ 148 
whfnp_mob3-projs_ 148

### CODE REVIEW

**Comment by lili2311** [comment]
Seems the projects are now no longer left aligned? Can we please add that back?

**Comment by VAO11** [comment]
Not sure what you mean? Are you referring to the images or text?

**Comment by lili2311** [comment]
The title and button are now centered, on the live site they are left aligned: http://www.womenhackfornonprofits.com/projects.html

**Comment by lili2311** [comment]
let me know once you pushed the changes so we can merge and deploy this :)

File: src/projects.html [hunk 1]
```diff
@@ -10,9 +10,11 @@ <h1 class="title-heading">Our Projects</h1>
 <section>
     <div class="container">
         <h2 class="section-heading">Completed Projects</h2>
-        <div class="project">
-            <div class="project-img fashrevwall"></div>
-            <div class="project-info">
+        <div class="row">
+            <div class="project-img fashrevwall col-xs-12 col-md-6">
+                <!--img src="" class="" alt="fashrevwall" /-->
```
Comment (lili2311) [inline src/projects.html]: please remove commented out code

**Review by lili2311 — CHANGES_REQUESTED** [review]

File: src/projects.html (hunk 1 above)
Comment (lili2311) [reply src/projects.html]: apart from that 👍  merge away

**Review by VAO11 — COMMENTED** [review]
Comments removed, Grunt files amended. Can this now be merged into Dev?

**Pull request merged**